client.go

```diff
@@ -472,3 +474,96 @@ func checkContainerRunning(id string) (types.Container, error) {
 
 	return container, nil
 }
+
+// Commits the container based on the given container ID to a new image
+// and runs "zypper lu" in it. Afterwards the image will be removed.
+func listUpdatesStoppedContainer(containerID string) (string, error) {
+	client := getDockerClient()
+	commitResp, err := client.ContainerCommit(types.ContainerCommitOptions{ContainerID: containerID})
```

Comment:
This `err` is being ignored. If this is intention, please call it `_` as that indicates more clearly that the error is being ignored. However I would recommend not ignoring errors -- at least log them if you're not going to bail.